Not compatible with handlebars 4.7.0+

Hi,

first of, I love your project, it's awesome.

I had to install it to a new project yesterday, and it didn't generate template with the emojis. 

For example, here is what was generated : 

```
# v2.0.5 (2020-02-06)

```

And, that's it. I looked into the logs and found this :

```
Handlebars: Access has been denied to resolve the property "ambulance" because it is not an "own property" of its parent.
You can add a runtime option to disable the check or this warning:
See https://handlebarsjs.com/api-reference/runtime-options.html#options-to-control-prototype-access for details
```

With a downgrade of handlebars in my lockfile, it worked again.

Can you please upgrade the dependancy level and add the corrects flags in order for this to work again on new projects ? :) 

Thanks again for your amazing tool =D

To reproduce the bug, just update the handlebar dependancy to at least 4.7.0 and run the tests, they will fail :

```
> ava test/**/*.test.js


⠧ analyze-commits › default config + common context w/ major updates

  4 passedHandlebars: Access has been denied to resolve the property "bug" because it is not an "own property" of its parent.
⠇ analyze-commits › default config + common context w/ major updates

  4 passedHandlebars: Access has been denied to resolve the property "sparkles" because it is not an "own property" of its parent.
⠋ analyze-commits › default config + common context w/ major updates

  4 passedHandlebars: Access has been denied to resolve the property "boom" because it is not an "own property" of its parent.

  4 tests failed

  generate-notes › default config + common context w/ patch updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v0.0.1](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v0.0.1) (2020-02-06)␊
    ␊
  + ## 🐛 Bug Fixes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + ␊
    `



  generate-notes › default config + common context w/ minor updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v0.1.0](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v0.1.0) (2020-02-06)␊
    ␊
  + ## ✨ New Features␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + ␊
  + ## 🐛 Bug Fixes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + ␊
    `



  generate-notes › default config + common context w/ major updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v1.0.0](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v1.0.0) (2020-02-06)␊
    ␊
  + ## ✨ New Features␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature ␊
  + ␊
  + ## 🐛 Bug Fixes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Fix bugs ␊
  + ␊
  + ## 💥 Breaking Changes␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Introduce a breaking change ␊
    `



  generate-notes › default config + WIP context w/ minor updates
    ℹ Release notes are generated.

  /home/beaussan/perso/git/semantic-release-gitmoji/test/integration/generate-notes.test.js:102

   101:   context.logger.log = t.log                                     
   102:   t.is(await generateNotes(pluginConfig, context), expectedNotes)
   103: }                                                                

  Difference:

    `# [v1.0.0](https://github.com/momocow/semantic-release-gitmoji/compare/v0.0.0...v1.0.0) (2020-02-06)␊
    ␊
  + ## ✨ New Features␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature (Issues: [`#1`](https://github.com/momocow/semantic-release-gitmoji/issues/1))␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  + ␊
  + - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Add a new feature (Issues: [`#2`](https://github.com/momocow/semantic-release-gitmoji/issues/2))␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  +     - [`commit_short`](https://github.com/momocow/semantic-release-gitmoji/commit/commit_short)  Work in progress␊
  + ␊
    `

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! semantic-release-gitmoji@1.3.2 test: `ava test/**/*.test.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the semantic-release-gitmoji@1.3.2 test script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/beaussan/.npm/_logs/2020-02-06T11_48_17_536Z-debug.log

```

--- a/lib/release-notes.js
+++ b/lib/release-notes.js
@@ -131,6 +131,8 @@ module.exports = class ReleaseNotes {
     return this._renderer({
       ...this._context,
       commits: proxifyCommitsMap(this._context.commits)
+    }, {
+      allowProtoPropertiesByDefault: true
     })
       .replace(/\n{3,}/g, '\n\n') // allow a blank line
   }

--- a/package.json
+++ b/package.json
@@ -51,7 +51,7 @@
     "debug": "^4.1.0",
     "emoji-regex": "^7.0.1",
     "git-url-parse": "^11.1.1",
-    "handlebars": "^4.1.2",
+    "handlebars": "^4.7.0",
     "issue-regex": "^2.0.0",
     "lodash.clonedeep": "^4.5.0",
     "lodash.mergewith": "^4.6.1",

